frontend/src/app/main/ui/workspace/tokens/form.cljs

```diff
@@ -313,10 +313,15 @@
                              (on-update-value-debounced value))))
         on-update-color (mf/use-fn
                          (mf/deps on-update-value-debounced)
-                         (fn [hex-value]
-                           (reset! value-ref hex-value)
-                           (set! (.-value (mf/ref-val value-input-ref)) hex-value)
-                           (on-update-value-debounced hex-value)))
+                         (fn [hex-value alpha]
+                           (let [color-value (if (= 1 alpha)
+                                               hex-value
+                                               (-> (tinycolor/valid-color hex-value)
+                                                   (tinycolor/set-alpha alpha)
+                                                   (tinycolor/->rgba-string)))]
+                             (reset! value-ref color-value)
+                             (set! (.-value (mf/ref-val value-input-ref)) color-value)
```

Comment:
Is better to use the dom.cljs name space and change that line for something like this
`(dom/set-value! node new-val)`